src/core/app.lua

```diff
@@ -20,6 +20,12 @@ local use_restart = false
 
 test_skipped_code = 43
 
+ingress_packet_drops = {
+   threshold = 100000,
+   wait = 20,
+   last_value = 0,
```

Comment:
In order to avoid allocations from the uint64 return from `:ingress_packet_drops()` from escaping, probably this should be `last_value = ffi.new('uint64_t[0]')`, and accesses should go through `ingress_packet_drops.last_value[0].`